src/viewer.rs

```diff
@@ -203,6 +200,7 @@ impl Viewer {
     }
     pub fn add_ground(
         &mut self,
```

Comment:
Just a comment: It seems that some member functions can become just functions now.